index.html

```diff
@@ -13,6 +13,7 @@
 </head>
 <body>
     <main>
+      <div class="bck">
```

Comment:
- [ ] As requested by the previous reviewer, kindly fix your design issues, to fix the size of this background, I recommend you add the property `width: 100%;` to the class `bck`. Please note that you might find new design bugs after you add this property, they will be easy to fix.